src/data-workspace/category-combo-table.js

```diff
@@ -47,7 +47,7 @@ export const CategoryComboTable = ({
 
     if (
         computedCategoryOptions.length !==
-        categoryCombo.categoryOptionCombos.length
+        categoryCombo.categoryOptionCombos?.length
```

Comment:
Yeah, it shouldn't really happen in practice. But it might happen if the category is somehow an `attribute`-categoryCombo.

Reply:
Alright, I'll create a jira issue for that so that it gets fixed on the BE side

Reply:
Nono, it's deliberate. But this is just to prevent the app from crashing if a attribute-combo would be selected, but that shouldn't really happen. This is mostly a relic from before the API was fixed, and default-catcombo did not return the `categoryOptionCombos`.

Reply:
I see! Could you add a comment to the code explaining what you just explained to me?

Reply:
It is kind of mentioned for the selector here: https://github.com/dhis2/data-entry-app/pull/50/files#diff-0f4a3b4e7a1af6e32767e349e7bdc6c920e1c9606a48dce49d0d00db87155bcfR45

Reply:
Ah, nice